RELEASES.md

```diff
@@ -1,3 +1,152 @@
+Version 1.45.0 (2020-07-16)
+==========================
+
+Language
+--------
+- [Float to int conversion using `as` has been defined as a saturating][71269]
+  This was previously undefined behaviour.
+- [`mem::Discriminant<T>` now uses `T`'s discriminant value instead always
+  using `u64`.][70705]
+- [Procedural macros can now be used in expression, pattern, and  statement
+  positions.][68717] This now means you can now use a procedural macro anywhere
+  you can use a declarative (`macro_rules!`) macros.
+
+Compiler
+--------
+- [You can now override indivdual target features through the `target-feature`
```

Comment:
indivdual

Reply:
Thanks for the fedback, in the future would you be able to leave nits as GitHub "suggestions"? as that makes it easier for me to add them. E.g.
```suggestion
- [You can now override individual target features through the `target-feature`
```